src/providers/actions-service/actions-service.ts

```diff
@@ -33,9 +32,17 @@ export default class ActionsService {
   fetchPaginatedActions(params: FetchActionsParams = {}): Observable<PaginatedResponse<Action>> {
     return this.http
       .get<any[]>(RESOURCE_PATH, { params, observe: 'response' })
-      .pipe(map(response => {
-        return new PaginatedResponse<Action>(response, parseApiAction);
-      }));
+      .pipe(map(response => new PaginatedResponse<Action>(response, parseApiAction)));
+  }
+
+  /**
+   * Fetchs one action, through its ID, from the BioPocket API.
+   * @param actionId The id of the action to fetch.
+   */
+  fetchAction(actionId: string): Observable<Action> {
+    return this.http
+      .get<Action>(`${RESOURCE_PATH}/${actionId}`)
```

Comment:
```suggestion
      .get(`${RESOURCE_PATH}/${actionId}`)
```

It's not an `Action` until it's passed through `parseApiAction`.